src/main/java/com/coveo/AbstractFMT.java

```diff
@@ -1,4 +1,4 @@
-package com.coveo;
+package com.spotify;
```

Comment:
Did you change just the package but didn't move the files?

Reply:
Damnit yes...